### PR TITLE
Styling Stage

### PR DESCRIPTION
**Pull request by Leandro-Barretoo**
In this milestone, I added the styles following a Zeplin guideline.
The live link will be added when the project is approved.
Thank you for the review.

### CODE REVIEW

File: src/components/BookInfo.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;400;600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@200;400;600&display=swap');
+
+.Info-Box {
+  flex: 3;
+}
+
+.Cat-Info {
+  display: block;
```
Comment (oxenprogrammer) [inline src/components/BookInfo.css:9]: **Changes Required**
I must say your CSS files are well organized and modular, bravo. However;
- [x] The CSS selector property `display` with the value of `block` is repeated multiple times. You want to follow best CSS practice of Don't Repeat Yourself. (DRY). Please let us refactor it by creating for instance a `.display-block` in your `.jsx` file and having a `.display-block { display: block}` in your CSS file. Please do this for all repeating Selector properties in all your CSS files.

File: src/components/BookInfo.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  padding: 2.5px 10px;
+  font-size: 0.975rem;
+  color: #121212;
+  opacity: 0.5;
+  font-weight: bold;
+}
+
+.Tit-Info {
+  display: block;
+  font-family: 'Roboto Slab', serif;
```
Comment (oxenprogrammer) [inline src/components/BookInfo.css]: **Changes Required**
I must say your CSS files are well organized and modular, bravo. However;
- [x] The CSS selector property `font-family` with the value of `Roboto Slab, serif` is repeated multiple times. You want to follow best CSS practice of Don't Repeat Yourself. (DRY). Please let us refactor it by creating for instance a `.family-roboto` in your `.jsx` file and having a `. family-robot{ font-family: 'Roboto Slab', serif;}` in your CSS file. Please do this for all repeating Selector properties in all your css files.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @Leandro-Barretoo , 

Good job so far!
I love your styling. Good effort making it look like the Zeplin design. 
I can see you're really putting effort to become a world-class software developer :+1:
There are some issues that you still need to address in order to move forward to the next project.
Trust me you got this! :muscle:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Consider the optional changes nice to have features and only work on them when you have satisfied the required changes_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by Leandro-Barretoo** [comment]
Thank you for the review, I applied the requested changes.

**Review by ZawadiSumba66 — APPROVED** [review]
### Hi @Leandro-Barretoo 👋 , 
### STATUS: APPROVED 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**